spec/rubocop/cop/naming/variable_number_spec.rb

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Naming::VariableNumber, :config do
-  let(:cop_config) { { 'CheckMethodNames' => true, 'CheckSymbols' => true } }
```

Comment:
I am not sure what this was intended to do or if it matters.

Reply:
In such case you need to drop the `:config` as well, right?

Reply:
It appears to be necessary in this case. I think it also must define `cop` because I see:

```
undefined local variable or method `cop' for ...
```